contracts/CommonNFT.sol

```diff
@@ -21,6 +23,14 @@ abstract contract CommonNFT is ERC721Enumerable {
         _;
     }
 
+    function setBaseURI(string memory baseURI_) public onlyOwner {
+        _baseURIextended = baseURI_;
+    }
```

Comment:
- it's better to mark this as `external` since this function has no internal use. (cleaner, cheaper)
- input paramter can be marked as `calldata` for a better use.
- it would be good to add tests for new functions.